submissions/sayckl/friends-app/script.js

```diff
@@ -0,0 +1,115 @@
+const cards = document.querySelector('.cards');
+const az = document.querySelector('.a-z');
+const za = document.querySelector('.z-a');
+const min = document.querySelector('.min');
+const max = document.querySelector('.max');
+const male = document.querySelector('.male');
+const female = document.querySelector('.female');
+const all = document.querySelector('.all');
+const reset = document.querySelector('.reset');
+let staticPeople;
+let people;
+
+const menu = document.querySelector('.menu-btn');
+const aside = document.querySelector('aside');
+menu.addEventListener('click', () => {
+  menu.classList.toggle('menu-btn_active');
+  aside.classList.toggle('show_aside');
+});
+
+fetch("https://randomuser.me/api/?results=40&nat=us")
+  .then(function(response) {
+    if (response.ok) {
+      return response.json();
+    }
+  })
+  .then(function(data) {
+     staticPeople = data.results;
+     people = staticPeople;
+     add(staticPeople);
+   });
+
+function add(people){
+  people.forEach(function(elem) {
+    capitals(elem);
+    const div = document.createElement('div');
+    div.className = 'card';
+    div.innerHTML = `<img src="${elem.picture.large}">
+    <div class="name">${elem.name.first} ${elem.name.last}</div>
```

Comment:
Maybe `span` of `p` will fit better? Like for text element

Reply:
changed div's to span's